deploy/olm-catalog/mobile-developer-console-operator/0.3.0/mobile-developer-console-operator.v0.3.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,270 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[{"apiVersion":"mdc.aerogear.org/v1alpha1","kind":"MobileDeveloperConsole","metadata":{"name":"example-mdc"},"spec":{"oAuthClientId":"mobile-developer-console","oAuthClientSecret":"foobar"}}]'
+    capabilities: Basic Install
+  name: mobile-developer-console-operator.v0.3.0
+  namespace: placeholder
```

Comment:
Is this okay? Should it be updated?

Reply:
Yeah I think this is fine, the namespace(s) should be set through the OLM, right?

Reply:
Yeah I was thinking that would be answer. Resolving.